### ISSUE TITLE
omething went wrong, exiting. Check /root/XenOrchestraInstallerUpdater/logs/xo-install.log-202402172323

### ISSUE THREAD

**Issue by davparker** [ronivay/XenOrchestraInstallerUpdater#210]
OS: Debian 12 
**Node.js version**: node -v  v18.19.1
**Yarn version**: yarn -v  1.22.19

**Server specs**
amount of vCPUs and RAM on the machine where you attempt to install
4 vCPUs   4 GB RAM

**Issue**
Running install
[info] xo-server and xo-web build takes quite a while. Grab a cup of coffee and lay back

[..] Running installation

[fail] Something went wrong, exiting. Check /root/XenOrchestraInstallerUpdater/logs/xo-install.log-202402172323 for more details and use rollback feature if needed

[fail] Removing /opt/xo/xo-builds/xen-orchestra-202402172323 because of failed installation.
[xo-install.log-202402172323.log](https://github.com/ronivay/XenOrchestraInstallerUpdater/files/14321283/xo-install.log-202402172323.log)

**Installation logfile**
Attach complete log as a file.

**Comment by davparker** [comment]
Out of curiosity I built an Ubuntu 20.04 server and tried to run the script there. It fails at the same point.

OS: Ubuntu 20.04
Yarn version:  yarn -v 1.22.19

Server specs
amount of vCPUs and RAM on the machine where you attempt to install
4 vCPUs 4 GB RAM

[info] xo-server and xo-web build takes quite a while. Grab a cup of coffee and lay back

[..] Running installation

[fail] Something went wrong, exiting. Check /home/david/XenOrchestraInstallerUpdater/logs/xo-install.log-202402182148 for more details and use rollback feature if needed

[fail] Removing /opt/xo/xo-builds/xen-orchestra-202402182148 because of failed installation.

**Comment by davparker** [comment]
Runnning the script the name resolution was trying to use ipv6 to connect. This was causing a failure. Weird, I could run apt install and install whatever software I needed.  Was the same on Ubuntu. It looks like my ISP recently began handing out IPv6 addresses. I had disabled passing IPv6 on my pfSense as my ISP was dragging out support for IPv6. For now I disabled IPv6 on the VM. The install completed successfully. It is always the simple things.
